packages/calypso-e2e/src/lib/components/editor-settings-sidebar-component.ts

```diff
@@ -14,6 +14,7 @@ const selectors = {
 		`${ sidebarParentSelector } .components-panel__body-toggle:has-text("${ sectionName }")`,
 	expandedSection: ( sectionName: EditorSidebarSection ) =>
 		`${ sidebarParentSelector } .is-opened .components-panel__body-toggle:has-text("${ sectionName }")`,
+	lastSection: `${ sidebarParentSelector } .components-panel__body:last-of-type`,
```

Comment:
AFAIR, the [`:last-of-type`](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-of-type) pseudo doesn't take the classname into consideration, so the above would be equivalent to 
```js
lastSection: `${ sidebarParentSelector } div:last-of-type`,
```
...which would work in this particular case as there doesn't seem to be any other child divs for that parent. Nevertheless, if we want to make this class-based, I think we need to use the [nth element selector](https://playwright.dev/docs/selectors/#n-th-element-selector):

```suggestion
	lastSection: `${ sidebarParentSelector } .components-panel__body >> nth=-1`,
```

Reply:
Ooo, great catch, thank you! I've updated that :)